src/components/Convert/routes/psm.ts

```diff
@@ -0,0 +1,80 @@
+import { BigNumber, ethers } from "ethers";
+import { HlpConfig, Network } from "../../..";
+import { PSM_GAS_LIMIT } from "../../../config/hlp";
+import { HPSM__factory } from "../../../contracts";
+import { ConvertQuoteInput, ConvertTransactionInput, Quote } from "../Convert";
+import { isTokenPegged } from "../isTokenPegged";
+import { PSM_WEIGHT, WeightInput } from "./weights";
+
+let transactionFeeCache: Record<Network, BigNumber | null> = {
```

Comment:
```suggestion
const transactionFeeCache: Record<Network, BigNumber | null> = {
```
This one can be const. It only has to be `let` if you re-assign the variable itself.

Reply:
done